### ISSUE TITLE
Font color for focused line in find-and-replace 

### ISSUE THREAD

**Issue by tschaub** [ignism/nucleus-dark-ui#66]
In the find and replace dialog, when input lines are focused, the background color changes.  It looks like the selector that sets the font color in this case doesn't apply if the "Use Regex" option is selected.

With no options, there is good contrast between the background and font color in the focused line (where I've typed "high contrast").

![high-contrast](https://cloud.githubusercontent.com/assets/41094/15839726/3c6728d2-2c02-11e6-83dc-226a0e145a07.png)

Here is the relevant markup for the screenshot above:

``` html
<span class="text plain null-grammar">high contrast</span>
```

When you select the "Use Regex" option, the font color for the focused line changes.  This results in much lower contrast between the background and font color.

![low-contrast](https://cloud.githubusercontent.com/assets/41094/15839737/4536cef4-2c02-11e6-8d7a-34db56139013.png)

And here is the relevant markup for the screenshot above:

``` html
<span class="source js regexp">low contrast</span>
```

I see this CSS applied when I do not have the "Use Regex" option selected.

``` css
atom-text-editor[mini].is-focused .text,
atom-text-editor[mini].is-focused::shadow .text {
  color #1d1f21;
}
```

I assume it would be enough to add selectors for the `source` class as well - though I don't know if this will consistently do the right thing.

The changes in this branch fix the issue for me: https://github.com/ignism/nucleus-dark-ui/compare/master...tschaub:search-text

Would you accept a pull request or suggest another way of solving this?

**Comment by ignism** [comment]
I will look into this, I noticed the font also changes, but doesnt change back if you turn regexp off.

**Comment by ignism** [comment]
should be fixed in patch 0.8.3, let me know if this is ok for you.

**Comment by tschaub** [comment]
Looks good to me.  Thanks for the great theme @ignism.
